Automattic/jetpack
# Fixes gulp language extract task async problem.

Fixes a problem with the language extracting task where it wouldn't properly signal an end.

#### Changes proposed in this Pull Request:

* Added a `return` statement to the `gulp` task so that Gulp knows when the task is over. It looks like this one broke sometime ago, but nobody noticed because it's not regularly run.
* Removed gulp tasks that merged POT files into one and converted it into a PHP file.
* Changed the jetpack-strings file generation back to "direct to PHP" mode to avoid unnecessary conversions.

#### Testing instructions:





* Make sure `yarn gulp languages:extract` runs properly, doesn't fail.
* Additionally `yarn build-languages` and `yarn build-production` should work too, no failures expected.

#### Proposed changelog entry
Not needed.
